backend/battle/views.py

```diff
@@ -43,11 +45,13 @@ class TeamView(LoginRequiredMixin, CreateView):
     form_class = TeamForm
     success_url = reverse_lazy("invite")
 
-    def get_initial(self):
-        obj_battle = get_object_or_404(Battle, pk=self.kwargs['pk'])
-        obj_trainer = self.request.user
-        self.initial = {"battle": obj_battle, "trainer": obj_trainer}
-        return self.initial
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        pokemons = Pokemon.objects.all()
+        context['pokemons'] = pokemons
+        context['battle'] = self.kwargs['pk']
```

Comment:
`battle_pk` is a better name for the context variable. `battle` implies an object.

Reply:
Sure, thank you again :)